### PR TITLE
Add error feedback on jenkins job updates

### PR DESCRIPTION
**Pull request by josvazg**
For HTTP errors you get output like:

```
backend_1  | Updating jobs...
backend_1  | There was an error contacting your Jenkins instance:
backend_1  | http://josvaz:**hidden*secret**@jenkins-webdev.nami/jenkins/api/json => 401 Unauthorized
```
```
backend_1  | Updating jobs...
backend_1  | There was an error contacting your Jenkins instance:
backend_1  | http://josvaz:**hidden*secret**@jenkins-webdev.nami/jenkin/api/json => 404 Not Found
```

For network errors that don't return a request object at all you get:
```
backend_1  | Updating jobs...
backend_1  | There was an error contacting your Jenkins instance:
backend_1  | No request data: Network error. Check your JENKINS_URL setting.
```

### CODE REVIEW

File: app/jobs.js [hunk 1]
```diff
@@ -99,4 +99,23 @@ const jobs = (jenkins, db) => {
   };
 };
 
+function dumpRequesError(r) {
```
Comment (Angelmmiguel) [inline app/jobs.js]: Use the new ES6 syntax for functions:

```
const dumpRequestError = (r) => {
  ... 
}
```

File: app/jobs.js [hunk 2]
```diff
@@ -99,4 +99,23 @@ const jobs = (jenkins, db) => {
...
+  if (r == null) {
+    console.error(`No request data: Network error. Check your JENKINS_URL setting.`)
+    return
+  }
+  uri = r.request.uri
+  url = `${uri.protocol}//${filterAuth(uri.auth)}${uri.hostname}${uri.path}`
+  console.error(`${url} => ${r.statusCode} ${r.statusMessage}`)
+}
+
+function filterAuth(auth) {
```
Comment (Angelmmiguel) [inline app/jobs.js]: Same here.

File: app/jobs.js [hunk 3]
```diff
@@ -99,4 +99,23 @@ const jobs = (jenkins, db) => {
   };
 };
 
+function dumpRequesError(r) {
+  console.error(`There was an error contacting your Jenkins instance:`);
+  if (r == null) {
+    console.error(`No request data: Network error. Check your JENKINS_URL setting.`)
```
Comment (Angelmmiguel) [inline app/jobs.js]: Add semicolons for the lines. Just for consistency

**Review by Angelmmiguel — CHANGES_REQUESTED** [review]
LGTM! Only some small changes

File: app/jobs.js (hunk 1 above)
Comment (josvazg) [reply app/jobs.js]: Will fix. But what is the benefit of this notation?
These are internal functions, so I don't expect them to be used externally.

Comment (Angelmmiguel) [reply app/jobs.js]: There are several advantages to use this notation. The first one is that the `this` variable is properly set inside them (scope). They are not focused on exporting :)

**Review by Angelmmiguel — APPROVED** [review]

**Pull request merged**